vkrunner/vr-script.h

```diff
@@ -36,6 +36,8 @@
 #include "vr-source.h"
 #include "vr-shader-stage.h"
 
+#define DEFAULT_TOLERANCE 0.01
+
```

Comment:
I think this should be moved into `vr-script.c` if it’s not needed elsewhere. If it is needed elsewhere it should be namespaced `VR_SCRIPT_DEFAULT_TOLERANCE`.